cmd/admin-user-policy.go

```diff
@@ -76,12 +78,28 @@ func mainAdminUserPolicy(ctx *cli.Context) error {
 		fatalIf(probe.NewError(e).Trace(args...), "Unable to fetch user policy document")
 	}
 
-	policies := strings.Split(user.PolicyName, ",")
-	for _, policy := range policies {
-		pinfo, e := getPolicyInfo(client, policy)
-		fatalIf(probe.NewError(e).Trace(args...), "Unable to fetch user policy document for policy "+policy)
-		fmt.Println(string(pinfo.Policy))
+	policyNames := strings.Split(user.PolicyName, ",")
+
+	var policies []policy.Policy
+	for _, policyName := range policyNames {
+		if policyName == "" {
+			continue
+		}
+		policyInfo, err := getPolicyInfo(client, policyName)
+		if err != nil {
+			fatalIf(probe.NewError(err).Trace(), "Unable to fetch user policy document for policy "+policyName)
+		}
+
+		var policyObj policy.Policy
+		if err := json.Unmarshal(policyInfo.Policy, &policyObj); err != nil {
+			fatalIf(probe.NewError(err).Trace(), "Unable to unmarshal policy")
+		}
+		policies = append(policies, policyObj)
 	}
 
+	mergedPolicy := policy.MergePolicies(policies...)
+	policyStr, _ := json.Marshal(mergedPolicy)
```

Comment:
```suggestion
	policyStr, _ := json.Marshal(mergedPolicy)
```

don't ignore errors or use `json.NewEncoder(os.Stdout).Encode(mergedPolicy)`

Reply:
This makes sense, thanks @harshavardhana